pom.xml

```diff
@@ -99,6 +99,8 @@
         <moneta.version>1.4.2</moneta.version>
         <owasp-java-html-sanitizer.version>20220608.1</owasp-java-html-sanitizer.version>
         <logstash.version>7.4</logstash.version>
+        <tomcat-embeded.version>10.1.15</tomcat-embeded.version>
```

Comment:
This will be out of sync with the spring boot parent version very soon. We have to find a different way to hand over the version defined by the parent.

Reply:
Sorry, I just found out how it is in different project to handle this more elegantly